src/pages/Registration/index.tsx

```diff
@@ -144,7 +142,7 @@ export default function Registration() : JSX.Element {
     username,
     password,
   } : ISignUpForm) : Promise<void> {
-    auth().createUserWithEmailAndPassword(email, password)
+    userAPI.createUserWithEmailAndPassword(email, password)
```

Comment:
Could be simpler like `Users.createAuth(email, password)`

Reply:
Fixed by new commit.